.github/workflows/plugin.yaml

```diff
@@ -25,7 +25,17 @@ jobs:
           go-version: '1.21.3'
           check-latest: true
 
+      - name: Run GoReleaser Snapshot
+        if: ${{ ! startsWith(github.ref, 'refs/tags/') }}
```

Comment:
is there a way in GHA to run on release only?

Reply:
yes, we can set the release created trigger for that
but will be another job